geocity/apps/forms/migrations/0023_add_form_quick_access_slug.py

```diff
@@ -0,0 +1,33 @@
+# Generated by Django 4.2.1 on 2023-07-26 12:57
+
+import re
+import uuid
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("forms", "0022_datamigration_form_and_field_api_name_and_input_type"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="form",
+            name="quick_access_slug",
+            field=models.TextField(
+                blank=True,
+                default=uuid.uuid4,
+                null=True,
+                validators=[
+                    django.core.validators.RegexValidator(
+                        re.compile("^[-a-zA-Z0-9_]+\\Z"),
+                        "Enter a valid “slug” consisting of letters, numbers, underscores or hyphens.",
```

Comment:
This is because I used [validate_slug](https://docs.djangoproject.com/en/4.2/ref/validators/#validate-slug), a django-provided validator which evaluates to the RegexValidator you can see in the migration. 

So, the validation text is defined in the django source, and worry not, since it is localized ([proof here](https://github.com/django/django/blob/main/django/core/validators.py#L259)) and appears in French if that is the browser language (I tested this).

Reply:
Thanks for the feedback!